modules/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,155 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
This is not an assembly

Reply:
Or at least, you can 't use assemblies in the modules folder

Reply:
Changed it. Thanks Aidan.